tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -207,6 +207,85 @@ test.describe( 'Nested Tabs tests', () => {
 		await expect( activeTab ).toHaveCSS( 'margin-bottom', '50px' );
 		await expect( lastTab ).toHaveCSS( 'margin-top', '25px' );
 	} );
+
+	test( 'Check that active and non-active tabs color changes on hover', async ( { page }, testInfo ) => {
+		// Arrange.
+		const wpAdmin = new WpAdminPage( page, testInfo );
+
+		await wpAdmin.setExperiments( {
```

Comment:
Pull latest update from your previous task.

Reply:
Done

Reply:
Can you please update all of the tests to Tzvi's `.setup()` method?